api/actions.json

```diff
@@ -56,19 +56,47 @@
 -----------------------------------------------------------------------------------------------------------------------------------
 
 # Request Product Operation (POST)
-# Endpoint: http://localhost:5001/local-parikrama/us-central1/api/api/actions/request
+# Endpoint: http://localhost:5001/local-parikrama/us-central1/api/api/actions/requestProduct
 #Request
 {
-	"toBranch": "MxoS2K8t8jT7MATniD4x",
-	"fromBranch": "R1vTnxSLWByLa7NocDDX",
-	"product": "milk",
-	"operationalQuantity": 5,
-	"comments": "urgent requirement"
+	"fromBranch": "MxoS2K8t8jT7MATniD4x",
+	"toBranch": "R1vTnxSLWByLa7NocDDX", //always headoffice
+	"fromBranchName": "sarjapur",
+	"toBranchName": "madiwala",
+	"productId": "3V0TKHqOC2GLzzNceFKs",
+	"productName": "bread",
+	"operationalQuantity": 10,
+	"note": "urgent requirement"
 }
 
-#Response (This transactionId is pendingTransactionId)
+
+#Response
+{
+    "pendingRequestsId": "gnVACYkIFM6vd8baebkA"
+}
+
+-----------------------------------------------------------------------------------------------------------------------------------
+-----------------------------------------------------------------------------------------------------------------------------------
+
+# Accept product request Operation (POST)
+# Endpoint: http://localhost:5001/local-parikrama/us-central1/api/api/actions/transferproduct
+#Request
+{
```

Comment:
qq.. Are you not getting user as input for accept operation?

Reply:
umm no. Getting it from pendingRequests of the toBranch